### ISSUE TITLE
FluidBufNNDSVD crashes server

### ISSUE THREAD

**Issue by tedmoore** [flucoma/flucoma-sc#110]
Code that recreates the crash:

```
(
b = Buffer.read(s,FluidFilesPath("Nicol-LoopE-M.wav"));
~bases = Buffer.new(s);
~activations = Buffer.new(s);
~resynth = Buffer.new(s);
)

//how many bases do I need to decompose the buffer with 90% accuracy
(
FluidBufNNDSVD.processBlocking(s, b, ~bases, ~activations, coverage: 0.9, method: 1,action:{
    "% bases".format(~bases.numChannels).postln;
});
)
```

**Comment by weefuzzy** [comment]
This was a problem with the sclang class not sending the max fft size, which is now needed. fixed by 52f6e38dfbfc107d29c72237a43cbe672963eaea

**Comment by tremblap** [comment]
indeed this is fixed
